plugins/modules/system/java_keystore.py

```diff
@@ -379,7 +511,21 @@ def main():
         add_file_common_args=spec.add_file_common_args,
     )
     module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C')
-    process_jks(module)
+
+    jks = JavaKeystore(module)
+
+    if jks.exists():
+        if module.params['force']:
+            jks.create()
+        else:
+            if jks.cert_changed():
+                jks.create()
```

Comment:
With boolean short circuiting this can be simplified to:
```suggestion
        if module.params['force'] or jks.cert_changed():
            jks.create()
```